docs/platform_operators/external-databases.md

```diff
@@ -73,65 +72,129 @@ As prerequisite, you need to execute the following steps to configure your postg
-    CCDB_PASSWORD=$(yq read "$VALUES_FILE" 'capi.database.password')
-    CCDB_NAME=$(yq read "$VALUES_FILE" 'capi.database.name')
-    UAADB_USERNAME=$(yq read "$VALUES_FILE" 'uaa.database.user')
-    UAADB_PASSWORD=$(yq read "$VALUES_FILE" 'uaa.database.password')
-    UAADB_NAME=$(yq read "$VALUES_FILE" 'uaa.database.name')
+    CCDB_USERNAME=$(yq read db-values.yml 'capi.database.user')
+    CCDB_PASSWORD=$(yq read db-values.yml 'capi.database.password')
+    CCDB_NAME=$(yq read db-values.yml 'capi.database.name')
+    UAADB_USERNAME=$(yq read db-values.yml 'uaa.database.user')
+    UAADB_PASSWORD=$(yq read db-values.yml 'uaa.database.password')
+    UAADB_NAME=$(yq read db-values.yml 'uaa.database.name')
     ...
     ```
 
 ## Installation of an internal database
 
 If both, capi and uaa, are configured to use an external database, no internal database will be deployed.
 
-## Example
+## Example with external RDS database
 
 In the following section, an external database is created using the bitnami postgresql helm chart. Please note that this setup is **not suitable for production environments**.
 
-1. Install postgresql using helm
+1. Create a RDS database. The following command will create a small database for development. Please adjust the settings to your requirements.
 
     ```bash
-    helm repo add bitnami https://charts.bitnami.com/bitnami
-    helm install postgresql bitnami/postgresql
+    export PGPASSWORD=<your password>
+    aws rds create-db-instance \
+        --engine postgres \
+        --db-instance-identifier cf-for-k8s \
+        --allocated-storage 20 \
+        --db-instance-class db.t2.micro \
+        --db-subnet-group default \
+        --master-username postgres \
+        --master-user-password "$PGPASSWORD" \
+        --backup-retention-period 7 \
+        --publicly-accessible
     ```
 
-1. Create configuration file `db-values.yaml`
+1. Wait until RDS database is ready
+
+    ```bash
+    aws rds wait db-instance-available --db-instance-identifier cf-for-k8s
+    ```
+
+1. Extract database hostname
+
+    ```bash
+    aws rds describe-db-instances --db-instance-identifier cf-for-k8s | jq -r '.DBInstances[0].Endpoint.Address'
+    ```
+
+1. Create configuration file `db-values.yml`. Please replace the `host`, `password` and `ca_cert` in this file accordingly. You can download the certificate from [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html)
 
     ```yaml
     #@data/values
     ---
     capi:
       database:
         adapter: postgres
-        host: postgresql.default.svc.cluster.local
+        host: cf-for-k8s.c4pknugnyzdd.eu-central-1.rds.amazonaws.com
         port: 5432
         user: capi_user
-        password: capi_password
+        password: <password for capi_user>
         name: capi_db
+        ca_cert: |
+          -----BEGIN CERTIFICATE-----
+          MIIEBjCCAu6gAwIBAgIJAMc0ZzaSUK51MA0GCSqGSIb3DQEBCwUAMIGPMQswCQYD
```

Comment:
is there value in including this full example cert? If not, I'd maybe replace the full thing with just `...` to shorten and simplify documentation

Reply:
OK